gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -186,8 +186,14 @@ static const ScriptingRefBase* GetScriptingRef(PyObject* obj) {
 		RuntimeError("Invalid Scripting reference, must have SCRIPT_GROUP attribute.");
 		return NULL;
 	}
-	ScriptingGroup_t group = PyString_AsString(attr);
+	PyObject * temp_bytes = PyUnicode_AsEncodedString(attr, "UTF-8", "strict");
 	Py_DecRef(attr);
+	if (!temp_bytes) {
+		RuntimeError("Invalid Scripting reference, failed to decode SCRIPT_GROUP attribute.");
+		return NULL;
```

Comment:
Use nullptr instead; we're slowly transitioning to it ever since we upgraded our minimum standard version to c++11.